src/queries/a11yState.ts

```diff
@@ -0,0 +1,68 @@
+import type { ReactTestInstance } from 'react-test-renderer';
+import { AccessibilityState } from 'react-native';
+import { makeQueries } from './makeQueries';
+import type {
+  FindAllByQuery,
+  FindByQuery,
+  GetAllByQuery,
+  GetByQuery,
+  QueryAllByQuery,
+  QueryByQuery,
+} from './makeQueries';
+
+export function matchObject<T extends Record<string, unknown>>(
+  prop: T | undefined,
+  matcher: T
+): boolean {
+  return prop
+    ? Object.keys(matcher).length !== 0 &&
+        Object.keys(prop).length !== 0 &&
+        !Object.keys(matcher).some((key) => prop[key] !== matcher[key])
+    : false;
+}
+
+const queryAllByA11yState = (
+  instance: ReactTestInstance
+): ((accessibilityStateKey: AccessibilityState) => Array<ReactTestInstance>) =>
+  function queryAllByDisplayValueFn(accessibilityStateKey) {
+    return instance.findAll(
+      (node) =>
+        typeof node.type === 'string' &&
+        matchObject(node.props.accessibilityState, accessibilityStateKey)
+    );
+  };
+
+const getMultipleError = (a11yState: AccessibilityState) =>
```

Comment:
```suggestion
const getMultipleError = (state: AccessibilityState) =>
```

Reply:
I'm curious, why state over a11yState? the type says it's an accessibility state, it's more specific